nilearn/experimental/surface/tests/test_maskers.py

```diff
@@ -79,14 +98,35 @@ def test_transform_inverse_transform(shape, make_mini_img, assert_img_equal):
 
 @pytest.mark.parametrize("shape", [(), (1,), (3,), (3, 2)])
 def test_transform_inverse_transform_with_mask(
-    shape, make_mini_img, assert_img_equal, mini_mask
+    surf_mesh, assert_img_equal, shape
 ):
-    img = make_mini_img(shape)
-    masker = SurfaceMasker(mini_mask).fit(img)
+    # make a sample image with data 1-4 on left part
+    # and 10-50 on right part
+    mesh = surf_mesh()
+    img_data = {}
+    for i, (key, val) in enumerate(mesh.parts.items()):
+        data_shape = (*shape, val.n_vertices)
+        data_part = (
+            np.arange(np.prod(data_shape)).reshape(data_shape) + 1.0
+        ) * 10**i
```

Comment:
I don't see why the values would be 1-4 and 10^-50 then ?

Reply:
It's not 10^-50, but between 10 to 50. But it would be easier to read if I just hard-code the values explicitly.

Reply:
Ok yes just the first sample would be between 1-4 (on left) and 10-50 (on right).

Reply:
updated the comment https://github.com/nilearn/nilearn/pull/4657/commits/bb9e9b1d0836f6ba6441e7876e79e33de366806b